test/integration/tests/4045-no-run-benchmarks/files/ChangeLog.md

```diff
@@ -0,0 +1,3 @@
+# Changelog for foff-bench-not-build
```

Comment:
We don't need this file in the repo, for just a test. Same for the LICENSE file and the readme

Reply:
👍